client/src/components/Board/Board.tsx

```diff
@@ -94,12 +94,20 @@ const Board: FC = (): JSX.Element => {
             <div className={classes.board} {...provided.droppableProps} ref={provided.innerRef}>
               <AddColumnWidget droppableId="addColumnLeft" showing={showingWidgetLeft} />
               <Grid container>
-                {boardColumns
-                  ? boardColumns.map((column, index) => (
-                      <Box key={column._id}>
-                        <BoardColumn column={column.cards} droppableId={column._id} name={column.name} index={index} />
-                      </Box>
-                    ))
+                {board.columns
+                  ? board.columns.map(
```

Comment:
You can use && operator for cleaner code
https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator

Reply:
I just changed boardColumns to boards.columns. I didn't see this improvement. but I can update that.